<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistForm</class>
 <widget class="QWidget" name="DistForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="load">
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Название модели авто</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enterModel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Год выпуска</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enterDate"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Пробег авто (км)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enterMileage"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="enterInput">
         <property name="text">
          <string>Добавить введенное</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editSelected">
         <property name="text">
          <string>Редактировать данные</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteSelected">
         <property name="text">
          <string>Удалить выбранное</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sortUp">
         <property name="text">
          <string>Сортировать по пробегу</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
